easybuild/easyconfigs/o/OrthoFinder/OrthoFinder-2.3.8-foss-2019b-Python-2.7.16.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'PythonPackage'
+
+name = 'OrthoFinder'
+version = '2.3.8'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/davidemms/OrthoFinder'
+description = "OrthoFinder is a fast, accurate and comprehensive platform for comparative genomics"
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+github_account = 'davidemms'
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
```

Comment:
@lexming Did you check whether `2.3.8.tar.gz` is different from the `OrthoFinder_source.tar.gz` source tarball that is also available? That could be a massaged copy of the sources...

Reply:
Yep, they are different. `2.3.8.tar.gz` is the tarball of the github tree, while `OrthoFinder_source.tar.gz` seems to be a stripped down release without the tests. The main problem with `OrthoFinder_source.tar.gz` is that it is missing the `setup.py` and hence, it is not useful for us.

Reply:
Gotcha, thanks for the clarification!